### PR TITLE
[Datafactory] Added SQL DW Copy Command support in Copy Activity

### PR DESCRIPTION
**Pull request by davidzhaoyue**

### CODE REVIEW

**Comment by isra-fel** [comment]
You forgot to check in new models such as `DWCopyCommandSettings.cs`

**Comment by davidzhaoyue** [comment]
@isra-fel , sorry for missing the two files, which have been added.

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure Data Factory Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Data Factory Resources.")]
-[assembly: AssemblyVersion("4.3.0.0")]
-[assembly: AssemblyFileVersion("4.4.0.0")]
+[assembly: AssemblyVersion("4.5.0.0")]
```
Comment (isra-fel) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Properties/AssemblyInfo.cs]: `AssemblyVersion` only needs to be updated when it's a major version change.
The version in csproj needs to be updated as well.
Thanks

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (davidzhaoyue) [reply sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Properties/AssemblyInfo.cs]: @isra-fel , updated.  Thanks!

**Review by isra-fel — APPROVED** [review]

**Pull request merged**